shell/src/main/java/org/apache/accumulo/shell/commands/FateCommand.java

```diff
@@ -130,107 +132,131 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
+
+  private void printTx(Shell shellState, AdminUtil<FateCommand> admin, ZooStore<FateCommand> zs,
+      ZooReaderWriter zk, ServiceLock.ServiceLockPath tableLocksPath, String[] args, CommandLine cl,
+      boolean printStatus) throws InterruptedException, KeeperException, IOException {
+    // Parse transaction ID filters for print display
+    Set<Long> filterTxid = null;
+    if (args.length >= 2) {
+      filterTxid = new HashSet<>(args.length);
+      for (int i = 1; i < args.length; i++) {
+        try {
+          Long val = parseTxid(args[i]);
+          filterTxid.add(val);
+        } catch (NumberFormatException nfe) {
+          // Failed to parse, will exit instead of displaying everything since the intention was
+          // to potentially filter some data
+          throw new RuntimeException("Invalid transaction ID format: " + args[i], nfe);
         }
       }
+    }
 
-      // Parse TStatus filters for print display
-      EnumSet<TStatus> filterStatus = null;
-      if (cl.hasOption(statusOption.getOpt())) {
-        filterStatus = EnumSet.noneOf(TStatus.class);
-        String[] tstat = cl.getOptionValues(statusOption.getOpt());
-        for (String element : tstat) {
-          try {
-            filterStatus.add(TStatus.valueOf(element));
-          } catch (IllegalArgumentException iae) {
-            System.out.printf("Invalid transaction status name: %s%n", element);
-            return 1;
-          }
+    // Parse TStatus filters for print display
+    EnumSet<TStatus> filterStatus = null;
+    if (printStatus) {
+      filterStatus = EnumSet.noneOf(TStatus.class);
+      String[] tstat = cl.getOptionValues(statusOption.getOpt());
+      for (String element : tstat) {
+        try {
+          filterStatus.add(TStatus.valueOf(element));
+        } catch (IllegalArgumentException iae) {
+          throw new RuntimeException("Invalid transaction status name: " + element, iae);
         }
       }
+    }
 
-      StringBuilder buf = new StringBuilder(8096);
-      Formatter fmt = new Formatter(buf);
-      admin.print(zs, zk, context.getZooKeeperRoot() + Constants.ZTABLE_LOCKS, fmt, filterTxid,
-          filterStatus);
-      shellState.printLines(Collections.singletonList(buf.toString()).iterator(),
-          !cl.hasOption(disablePaginationOpt.getOpt()));
-    } else if ("dump".equals(cmd)) {
-      List<Long> txids;
+    StringBuilder buf = new StringBuilder(8096);
+    Formatter fmt = new Formatter(buf);
+    admin.print(zs, zk, tableLocksPath, fmt, filterTxid, filterStatus);
+    shellState.printLines(Collections.singletonList(buf.toString()).iterator(),
+        !cl.hasOption(disablePaginationOpt.getOpt()));
+  }
 
-      if (args.length == 1) {
-        txids = zs.list();
+  private boolean deleteTx(AdminUtil<FateCommand> admin, ZooStore<FateCommand> zs,
+      ZooReaderWriter zk, ServiceLockPath zLockManagerPath, String[] args)
+      throws InterruptedException, KeeperException {
+    boolean success = true;
```

Comment:
This looks like it's being used as a constant to represent SUCCESS, and not the actual variable value of whether or not it succeeded. As such, it should probably be final, and maybe a static final constant, all upper-case, so it doesn't look like a variable containing a result.

So, something like:

```java
  final SUCCESS = true;
  // ...
    return !SUCCESS;
  // ...
  return SUCCESS;
```

However, as a developer looking at this code, I'm not really sure what this is doing unless I look at the assignments to see what value is contained in SUCCESS. So, it'd probably just be cleaner to directly return `true` and `false`, rather than negating a variable or a constant whose value is unknown to the developer until they go look.